src/frontend/src/btc/stores/btc-transactions.store.ts

```diff
@@ -0,0 +1,41 @@
+import type { BtcTransactionUi } from '$btc/types/btc';
+import { initCertifiedStore, type CertifiedStore } from '$lib/stores/certified.store';
+import type { CertifiedData } from '$lib/types/store';
+import type { TokenId } from '$lib/types/token';
+import { nonNullish } from '@dfinity/utils';
+
+export type BtcCertifiedTransaction = CertifiedData<BtcTransactionUi>;
+
+export type BtcTransactionsData = BtcCertifiedTransaction[];
+
+export interface BtcTransactionsStore extends CertifiedStore<BtcTransactionsData> {
+	prepend: (params: { tokenId: TokenId; transactions: BtcCertifiedTransaction[] }) => void;
```

Comment:
No append and no cleanUp?

Reply:
It now uses a comment store template, so both append and cleanUp are a part of btc transactions store.